widget/ui/src/components/ExpandedRoute/ExpandedRoute.tsx

```diff
@@ -0,0 +1,130 @@
+import type { PropTypes } from './ExpandedRoute.types';
+
+import React from 'react';
+
+import { ChainToken } from '../ChainToken';
+import { Image } from '../common';
+import { Divider } from '../Divider';
+import { QuoteCost } from '../QuoteCost';
+import { TagRoute } from '../TagRoute';
+import { Tooltip } from '../Tooltip';
+import { Typography } from '../Typography';
+
+import {
+  FromToken,
+  lastStepStyle,
+  LeftSection,
+  OutputSection,
+  RightSection,
+  RouteContainer,
+  RouteHeader,
+  Step,
+  Steps,
+  StyledPriceImpact,
+  SwapperContainer,
+  TagsContainer,
+  VerticalLine,
+} from './ExpandedRoute.styles';
+
+export function ExpandedRoute(props: PropTypes) {
```

Comment:
Please double check if you handle step warning/error case.

<img width="307" alt="Screenshot 2024-02-20 at 10 37 49" src="https://github.com/rango-exchange/rango-client/assets/99180934/2a9b9a49-cb23-4145-8981-9607e24e29a3">

<img width="521" alt="Screenshot 2024-02-20 at 10 37 31" src="https://github.com/rango-exchange/rango-client/assets/99180934/375f0f30-92ce-4c3b-bc83-3bf20bc658de">